packages/vscode-extension/src/loggers/VSConsoleLogger.ts

```diff
@@ -14,6 +14,7 @@ const INVALID_JAR_FILE_MESSAGE = 'Invalid or corrupt jarfile';
 export default class VSConsoleLogger extends ConsoleLogger {
 	private _executionPath?: string;
 	private _addExecutionDetailsToLog: boolean = false;
+	private _hiddeProjectFolerName: boolean = false;
```

Comment:
Typo in "_hiddeProjectFolerName". Should this be "_hideProjectFolderName"?

And since this is TypeScript, I don't think we need to prefix private variables with underscores ("_") anymore? What do you think?